tests/Oryx.RuntimeImage.Tests/NodeImagesTest.cs

```diff
@@ -393,9 +393,10 @@ public async Task GeneratesScript_CanRun_AppInsightsModule_Found(string nodeVers
                 assertAction: async () =>
                 {
                     var data = await _httpClient.GetStringAsync($"http://localhost:{hostPort}/");
-                    Assert.Contains("Say It Again", data);
+                    Assert.Contains("Hello World from express!", data);
```

Comment:
how do we know for sure that appinsights module loaded correctly? Should we try accessing any appinsights api from within the app and try doing something with it...if all is well we should not see any errors. Thoughts?

Reply:
so we have an env var NODE_OPTIONS, where we add the appinsightsloader.js. Content of this loader file is basically setup appinsights by calling "appInsights.setup().start()". So in a way we are doing this.